tests/test_client.py

```diff
@@ -571,4 +679,35 @@ def test_user_can_view_unpublished_judgments_false(self):
             client.eval.return_value.text = 'false'
 
             result = client.user_can_view_unpublished_judgments("laura")
+            self.assertEqual(result, False)
+
+    def test_verify_show_unpublished_user_cannot_view_unpublished_and_show_unpublished_true(self):
```

Comment:
I've finally parsed these test names and I think it might be worth rewording them:

something like:
test_verifiy_forbids_unpublished_if_unauthorised_and_asks
test_verify_allows_unpublished_if_authorised_and_asks
test_verify_forbids_unpublished_if_unauthorised_and_doesnt_ask
test_verify_forbids_unpublished_if_authorised_and_doesnt_ask

Reply:
Definately worth looking into how we do contexts for these :)